### DISCUSSION TITLE
Authenticated users cannot create rows on a table

### DISCUSSION THREAD

**Discussion by bitttttten** [supabase/supabase#28791] [Questions]
I am using `Authentication` (https://supabase.com/dashboard/project/{id}/auth/users) to create users.

I want all these users to be able to create rows on a certain table. so I run

```sql
create policy "Enable update for authenticated users"
on "public"."mytable"
for UPDATE
to authenticated
with check (true);
```

However when I run something like

```ts
const result = await supabase.from("mytable").insert({..});
```

I always get back

```json
{code: '42501', details: null, hint: null, message: 'permission denied for table mytable'}
```

What am I doing wrong? I have RLS enabled with the above SQL command ran. I only ever make things in the supabase UI. Is there something else I need to run on my table? Or what am I missing?

**Comment by GaryAustin1** [comment]
Should  change update to insert.  If you had a .select() to that insert then you also need to meet select policy.

```
create policy "Enable insert for authenticated users"
on "public"."mytable"
for INSERT
to authenticated
with check (true);
```
